plugins/package-curation-providers/api/src/main/kotlin/SimplePackageCurationProvider.kt

```diff
@@ -21,11 +21,17 @@ package org.ossreviewtoolkit.plugins.packagecurationproviders.api
 
 import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.PackageCuration
+import org.ossreviewtoolkit.plugins.api.PluginDescriptor
 
 /**
  * A [PackageCurationProvider] that provides the specified [packageCurations].
  */
-open class SimplePackageCurationProvider(val packageCurations: List<PackageCuration>) : PackageCurationProvider {
+open class SimplePackageCurationProvider(
+    override val descriptor: PluginDescriptor,
+    val packageCurations: List<PackageCuration>
+) : PackageCurationProvider {
+    constructor(packageCurations: List<PackageCuration>) : this(PluginDescriptor.EMPTY, packageCurations)
```

Comment:
Without a deeper understanding of the code, it seems awkward to ever use `PluginDescriptor.EMPTY` in non-test code (which BTW the property's description says it's meant for). Could you please add documentation to this constructor explaining its use-case, and why it's ok to have an empty plugin descriptor here?

Reply:
I have removed the constant and replaced it:

For `SimplePackageCurationProvider` I have added constant descriptor, as this is no plugin on its own there is no generated one.

For `FilePackageCurationProvider` I am using `FilePackageCurationProviderFactory().descriptor`. I will check after this PR if I can make this a constant so that it is not required to create an instance of the factory.